index.js

```diff
@@ -190,3 +190,130 @@ const titleCase = str => {
     })
     .join(" ");
 };
+
+//Task 26
+
+const getAverage = marks => {
+  const allValues = marks.reduce((accum, curVal) => accum + curVal);
+  return Math.round(allValues / marks.length);
+};
+
+//Task 27
+
+class Human {
+  constructor() {}
+}
+class Man extends Human {
+  sex = "male";
+  constructor(name) {
+    super();
+    this.name = name;
+  }
+}
+class Woman extends Human {
+  sex = "female";
+  constructor(name) {
+    super();
+    this.name = name;
+  }
+}
+
+class God {
+  constructor() {}
+  static create(manName, womanName) {
+    let man = new Man();
+    man.name = manName;
+    let woman = new Woman();
+    woman.name = womanName;
+    const arr = [];
+    arr.push(man);
+    arr.push(woman);
+    return arr;
+  }
+}
+
+//Task 28
+
+const getVowelCount = str => {
+  let volwesCount = 0;
+  let internArr = str.split("");
+  for (let i = 0; i < internArr.length; i++) {
+    if (
+      internArr[i] === "a" ||
+      internArr[i] === "e" ||
+      internArr[i] === "i" ||
+      internArr[i] === "o" ||
+      internArr[i] === "u"
```

Comment:
too much for `if` condition.  Please create additional array with characters that are needed to be found: const neededCharacters = ['a', 'e', ....]. Then filter initial array with `includes()` method and return length of filtered array